metadata/energy/oil_gas_wells.md

```diff
@@ -0,0 +1,77 @@
+# Title
+
+Utah Oil and Gas Wells
+
+## ID
+
+303a22e9-0dc0-4e5e-9053-d671321900e9
+
+## Brief Summary
+
+Surface location of oil and gas wells from the Department of Natural Resources, Oil, Gas, and Mining Division.
+
+## Summary
+
+This dataset contains point locations of oil and gas wells from the Oil, Gas, and Mining Division (DOGM) at the Utah Department of Natural Resources. Points contain attributes such as the API code, well and company name, and well type and status.
+
+## Description
+
+### What is the dataset?
+
+Oil and gas wells in Utah are managed by the Utah Department of Natural Resources [Oil, Gas and Mining Division](https://ogm.utah.gov/). This dataset contains all oil and gas wells in Utah, including those being actively drilled, still in the permit process, and those that have been abandoned.
+
+### What is the purpose of the dataset?
+
+This dataset provides the opportunity for the Division to collect more descriptive down-hole information for each wellbore permitted and drilled.
+
+<!--- This is from the original metadata. Do we still see this dataset as an opportunity to collect more descriptive information, or does this dataset have a different purpose now? --->
```

Comment:
I'm not sure where this purpose came from. I would say that it's purpose is to show the surface locations for wells. We do not participate in any data collection for this dataset.